spec/models/custom_value/bool_strategy_spec.rb

```diff
@@ -85,6 +85,16 @@
       let(:value) { nil }
       it { is_expected.to be_nil }
     end
+
+    context 'value is true' do
+      let(:value) { true }
+      it { is_expected.to eql(true) }
```

Comment:
@NobodysNightmare since `true` is a singleton object in Ruby, you can use `be true` and `be false` here.